bin/celeste.jl

```diff
@@ -27,7 +33,175 @@ field, using only that field.
 The `score-field` subcommand is not yet implemented for the new API.
 """
 
+"""
+An area of the sky subtended by `ramin`, `ramax`, `decmin`, and `decmax`.
+"""
+type SkyArea
+    ramin::Float64
+    ramax::Float64
+    decmin::Float64
+    decmax::Float64
+    nra::Int64
+    ndec::Int64
+end
+
+"""
+Given a SkyArea that is to be divided into `skya.nra` x `skya.ndec` patches,
+return the `i`th patch. `i` is a linear index between 1 and
+`skya.nra * skya.ndec`.
+
+This function assumes a cartesian (rather than spherical) coordinate system!
+"""
+function sky_subarea(skya::SkyArea, i)
```

Comment:
Just my opinion, but I think it's good style to give functions verb names and variables noun names.